src/item.cpp

```diff
@@ -1744,7 +1766,7 @@ std::string item::tname( unsigned int quantity, bool with_prefix ) const
     }
 
     if (has_flag("FIT")) {
-        ret << _(" (fits)");
+        ret << _(" (fits) ");
```

Comment:
I meant the splitting of `} else {` onto 3 lines, including an empty one (new code line 1654). It looks as if you wanted to add something there.